docs/azure_arc_jumpstart/azure_arc_k8s/day2/cluster_api/cluster_api_gitops_basic/_index.md

```diff
@@ -0,0 +1,206 @@
+---
+type: docs
+title: "Deploy GitOps configurations and perform basic GitOps flow on Cluster API as an Azure Arc Connected Cluster"
+linkTitle: "Deploy GitOps configurations and perform basic GitOps flow on Cluster API as an Azure Arc Connected Cluster"
+weight: 4
+description: >
+---
+
+## Deploy GitOps configurations and perform basic GitOps flow on Cluster API as an Azure Arc Connected Cluster
+
+The following README will guide you on how to create GitOps configuration on a Cluster API which is projected as an Azure Arc connected cluster resource.
+
+In this guide, you will deploy & attach GitOps configuration to your cluster which will also include deploying an "Hello World" Azure Arc web application on your Kubernetes cluster. By doing so, you will be able to make real-time changes to the application and show how the GitOps flow takes effect.
+
```

Comment:
would be good to add one line on the fact that this is based on Flux and point to their overview page. nothing to fancy, just as an intro for familiarity

Reply:
Added the text from ther docs, also added the link